### ISSUE TITLE
add Industrial History walking tour layer

### ISSUE THREAD

**Issue by clhenrick** [bowon/demo_july10#35]
- call layers `industrialhistory` & `industrialhistory_l`

**Comment by clhenrick** [comment]
@bowon do you have icon(s) for this layer's button(s)? I'm assuming that there will only be one button to turn on both the points and lines?

**Comment by bowon** [comment]
Yes, Industrial history and polluted and remediated sites will call two layers at once. Please use the current Polluted sites button for the new "polluted and remediated sites" interaction. 

Bowon

**Comment by clhenrick** [comment]
@bowon did the names of these layers change in cartodb? Please confirm.

**Comment by bowon** [comment]
It updated as industrialhistory_points and industrialhistory_lines that was previously industrialhistory and industrialhistory_l. If something isn't responsing correctly, please check those names.

Bowon

**Comment by clhenrick** [comment]
@bowon please tell me when you change the names of tables! I just spent the past hour trying to debug this. If you change a table name, notify me.

**Comment by bowon** [comment]
Just after you sent me the email. it supposed to be OK now. calm down. 

Bowon

**Comment by bowon** [comment]
I feel like this issue is related to the interactivity section of industrial history visualization, rather than sql or cartocss.
I am reorganizing layers.js now. Some of interactivity sections should be corrected that found yesterday. 
I will update you the result as soon as possible, possibly in tomorrow. 

Bowon

**Comment by clhenrick** [comment]
@bowon please leave all things related to the map & data layers interaction to me. Just focus on the UI updates for now, thanks.

**Comment by clhenrick** [comment]
Not sure when CartoDB will fix this CartoCSS / CartoDB.JS bug so we may have to find a work around for labeling the points. One option is to custom render SVG markers.
